word-builder/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <script>
+      var myWord = "supercalifragilisticexpialidocious";
+      function wordBuilder(wordToSpell) {
+        builtWord = [];
```

Comment:
You are missing your var keyword here, which will make this an auto global variable. If you are making variables in a function, make sure you keep them in that function's scope! 😁